doc/ctsm-docs_container/Dockerfile

```diff
@@ -28,4 +28,5 @@ WORKDIR /home/user
 CMD ["/bin/bash", "-l"]
 
 LABEL org.opencontainers.image.title="Container for building CTSM documentation"
-LABEL org.opencontainers.image.source=https://github.com/ESCOMP/CTSM
\ No newline at end of file
+LABEL org.opencontainers.image.source=https://github.com/ESCOMP/CTSM
+LABEL org.opencontainers.image.version="v1.0.2"
```

Comment:
I'm not going to update the version used in doc-builder because the new v1.0.2 should be identical to the old v1.0.1 that doc-builder uses.